Makefile

```diff
@@ -42,17 +45,28 @@ bin/%:
 generate:
 	@hack/codegen.sh
 
-deploy: kind-load-manager
+deploy:
+	# We need to make sure the namespace is created before we apply any namespace-scoped configurations into cluster.
+	kubectl create namespace bulward-system
+	$(MAKE) deploy-manager
+	$(MAKE) deploy-apiserver
+
+deploy-manager: kind-load-manager
 	cd config/manager/manager && kustomize edit set image manager=${IMAGE_ORG}/bulward-manager:${VERSION}
 	kustomize build config/manager/default | kubectl apply -f -
 
+deploy-apiserver: kind-load-apiserver
+	cd config/apiserver/manager && kustomize edit set image manager=${IMAGE_ORG}/bulward-apiserver:${VERSION}
+	kustomize build config/apiserver/default | kubectl apply -f -
+	kubectl apply -f config/apiserver/rbac/extension_apiserver_auth_role_binding.yaml
```

Comment:
We need to apply this rolebinding manually, since kustmize will override the namespace of this rolebinding which we don't want to.